contrib/babelfishpg_tds/src/backend/tds/tdstypeio.c

```diff
@@ -1963,6 +1979,49 @@ TdsRecvTypeDatetime2(const char *message, const ParameterToken token)
 	return result;
 }
 
+/* -------------------------------
+ * TdsRecvTypeGeometry - converts external binary format to
+ * Geometry data type
+ * --------------------------------
+ */
+Datum
+TdsRecvTypeGeometry(const char *message, const ParameterToken token)
+{
+	Datum result = 0; 
+
+	//Decode binary and convert if needed 
+	StringInfo	buf = TdsGetStringInfoBufferFromToken(message, token);
+
+	//Return in Datum val
+
+	ereport(ERROR,
```

Comment:
so we will always throw an error on Recv?

Reply:
It is currently a placeholder function, So just copied the template,
although added a comment for placeholder function, will addressing this in subsequent PR's